parlai/agents/bart/bart.py

```diff
@@ -82,9 +82,9 @@ def _initialize_bart(self, opt: Opt) -> Opt:
         :return opt:
             return opt with BART-specific args.
         """
+        init_model, _ = self._get_init_model(opt, None)
         if not opt.get('converting') and (
-            opt.get('init_model') is None
-            or not PathManager.exists(opt.get('init_model', ''))
+            init_model is None or not PathManager.exists(init_model, '')
```

Comment:
sorry wait this is wrong you need `PathManager.exists(init_model)`

Reply:
ah oops i committed this fix but forgot to push